functions_and_scope_lab.js

```diff
@@ -0,0 +1,285 @@
+console.log(`# Function Exercises`);
+
+
+//## Question 1
```

Comment:
Good job overall. I like clean codes that you wrote here with ternary and implicit returns. Keep it up!